ledger-service/http-json-oracle/BUILD.bazel

```diff
@@ -0,0 +1,90 @@
+# Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+load(
+    "//bazel_tools:scala.bzl",
+    "da_scala_binary",
+    "da_scala_library",
+    "da_scala_test",
+    "lf_scalacopts",
+)
+load("@bazel_skylib//rules:common_settings.bzl", "string_flag")
+load("@scala_version//:index.bzl", "scala_version_suffix")
+
+hj_scalacopts = lf_scalacopts + [
+    "-P:wartremover:traverser:org.wartremover.warts.NonUnitStatements",
+]
+
+string_flag(
+    name = "oracle_testing",
+    build_setting_default = "no",
+)
+
+config_setting(
+    name = "oracle_available",
+    flag_values = {
+        ":oracle_testing": "yes",
+    },
+)
+
+config_setting(
+    name = "oracle_unavailable",
+    flag_values = {
+        ":oracle_testing": "no",
+    },
+)
+
+# ERROR: .../daml/ledger-service/http-json-oracle/BUILD.bazel:33:14: no such target
+# '@platforms//:incompatible': target 'incompatible' not declared in package '' defined by
+# /private/var/tmp/.../85ba8c006f4674ac1f1213a18c0df36a/external/platforms/BUILD and
+# referenced by '//ledger-service/http-json-oracle:integration-tests'
+constraint_setting(name = "incompatible_setting")
+
+constraint_value(
+    name = "incompatible",
+    constraint_setting = ":incompatible_setting",
+)
```

Comment:
Our dependency on `@platforms` may not be recent enough. It should be present [here](https://github.com/bazelbuild/platforms/blob/d87d2b43046778da7b09d032f49a0028f3b4d321/BUILD#L27).

Reply:
I assume we have to upgrade Bazel to upgrade this.

Reply:
We can upgrade only the `platforms` dependency. Done in https://github.com/digital-asset/daml/pull/8869. I’m fine with merging this in the current state though and we can clean it up once #8869 is merged.